src/_components/OurManualData.js

```diff
@@ -0,0 +1,42 @@
+export const data = [
```

Comment:
I would move this to a markdown file similar to how we do it on the Blueprint website as opposed to a data.js. This way they can edit any content for a page all in one place